app/views/admin/leagues/_form.html.erb

```diff
@@ -0,0 +1,33 @@
+<%= form_for @league, url:  yield(:link) do |f| %>
```

Comment:
có cần phải sử dụng url không?

Reply:
Xem cách dùng khác với form for nhé

Reply:
vì em sử dụng 2 controller admin và user khác nhau nên em sử dụng url để nó bắt đúng cái admin mình muốn